picard/util/tags.py

```diff
@@ -278,14 +297,19 @@ def _base_description(self, tagname):
         else:
             return f"<p><em>%{name}%</em></p>{content}"
 
+    def _add_sections(self, item, include_sections):
+        # Note: format has to be translatable, for languages not using left-to-right for example
```

Comment:
@rdswift this is something we need to take care of as much as possible, we should be able to display correctly formatted tooltips even in languages not using left to right direction, something like "Note: text" would be "txet :etoN" so the order of title and text and the position of the separator (":") here have to be translatable.
Hence this approach (basically we make the format translatable along the strings displayed by it).